static/src/javascripts/projects/commercial/modules/prebid/slot-config.spec.js

```diff
@@ -78,6 +78,7 @@ describe('getSlots', () => {
 
     test('should return the correct slots at breakpoint M for US including mobile sticky slot', () => {
         getBreakpointKey.mockReturnValue('M');
+        config.set('switches.mobileStickyPrebid', true);
```

Comment:
Seems like this config.set() should be inside beforeEach since you never set it to false.

Reply:
Yea good remark although tests are still valid since the condition only checks for true and when it will be `undefined` wills not run